### ISSUE TITLE
Feature request: Improvement for in-command help

### ISSUE THREAD

**Issue by tillig** [pb33f/openapi-changes#35]
If I run `openapi-changes summary` it shows me a few nice examples of how to use it, but it neither has a link to the online docs showing the flags (`-n`, `--no-logo`, etc.) nor any inline help showing the potential options.

It would be nice if:

- Running `openapi-changes -?` or `openapi-changes --help` would display a fairly standard help dialog, even if it was the same as just running it without any switches. It's not obvious on how to get help.
- Running `openapi-changes summary -?` would not only show the examples, but the list of available flags.

I think the colorful UI is pretty cool, but from an online help perspective, I think some of the classic help mechanisms like other commands (`git`, `kubectl`, etc.) have where the options are explained in a complete list might be interesting.

**Comment by daveshanley** [comment]
I agree, the experience could use some improvement.

**Comment by daveshanley** [comment]
Just checking that you did use the ` --help` flag to see the more traditional help views that are available? I know it needs better docs, but there should be an good bit of out of the box help already.

**Comment by tillig** [comment]
No, I admit I missed that. I was somewhat focused on the `summary` command at the time, so when I ran the original command without that

`openapi-changes`

and then it told me "I have a few options" but didn't mention anything about `--help` I made the [obviously incorrect] assumption that it was directing me to type the command to get help, sooooo

`openapi-changes summary`

Hey, this is nice, more colorful help. Since I'm getting help this way, this must be _the only way_ to get help.

Again, bad assumption on my part. Showing two different help screens for the same thing wasn't something that would ever have occurred to me, so I didn't try other things.
